Geometry/CMSCommonData/python/cmsExtendedGeometry2023devXML_cfi.py

```diff
@@ -22,37 +22,37 @@
         'Geometry/CMSCommonData/data/cavern.xml',
         'Geometry/TrackerCommonData/data/PhaseII/trackerParameters.xml',
         'Geometry/TrackerCommonData/data/pixfwdCommon.xml',
-	'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdMaterials.xml',
-	'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdCylinder.xml', 
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwd.xml', 
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdDisks.xml', 
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdInnerDisk1.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdInnerDisk2.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdInnerDisk3.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdInnerDisk4.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdInnerDisk5.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdInnerDisk6.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdInnerDisk7.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdOuterDisk1.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdOuterDisk2.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdOuterDisk3.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdOuterDisk4.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdOuterDisk5.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdOuterDisk6.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdOuterDisk7.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdOuterDisk8.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdOuterDisk9.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdOuterDisk10.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade1.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade2.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade3.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade4.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade5.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade6.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade7.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade8.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade9.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade10.xml',
+	'Geometry/TrackerCommonData/data/PhaseII/FlatTracker/pixfwdMaterials.xml',
```

Comment:
Can fix it but need to take care of other things first (HLT + PF related).
If you have time for a PR to this PR I'll merge it.



On Mon, Apr 18, 2016 at 1:33 PM, Kevin Pedro <notifications@github.com>
wrote:

> In Geometry/CMSCommonData/python/cmsExtendedGeometry2023devXML_cfi.py
> <https://github.com/cms-sw/cmssw/pull/14119#discussion_r60043040>:
>
> > -        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdOuterDisk6.xml',
> > -        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdOuterDisk7.xml',
> > -        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdOuterDisk8.xml',
> > -        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdOuterDisk9.xml',
> > -        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdOuterDisk10.xml',
> > -        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade1.xml',
> > -        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade2.xml',
> > -        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade3.xml',
> > -        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade4.xml',
> > -        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade5.xml',
> > -        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade6.xml',
> > -        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade7.xml',
> > -        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade8.xml',
> > -        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade9.xml',
> > -        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade10.xml',
> > +	'Geometry/TrackerCommonData/data/PhaseII/FlatTracker/pixfwdMaterials.xml',
>
> @ianna <https://github.com/ianna> - I assumed this was a rebase issue
> that @lgray <https://github.com/lgray> would take care of soon.
>
> I personally would prefer to rename 2023dev to 2023tilted, but let's keep
> the standard 2023 with the flat tracker as just the "default" scenario.
> However, such renaming has to be propagated to the upgrade matrix
> workflows, so maybe we should hold off for this PR.
>
> In general, we need a better way to organize the Phase2 variations. Right
> now tilted tracker is the only major case, but I'm sure there will be more
> to consider soon.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/14119/files/e6885ff5bab026b1a39e746b3969ff0f6018d88c#r60043040>
>